packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -2573,7 +2573,9 @@ export function attach(
 
   function getUpdatersList(root): Array<SerializedElement> | null {
     return root.memoizedUpdaters != null
-      ? Array.from(root.memoizedUpdaters).map(fiberToSerializedElement)
+      ? Array.from(root.memoizedUpdaters)
+          .filter(fiber => getFiberIDUnsafe(fiber) !== null)
+          .map(fiberToSerializedElement)
```

Comment:
This change should fix this bug in DevTools for older versions of React too 👍🏼  Nice.